setup.py

```diff
@@ -17,6 +17,7 @@
     "sympy",
     "transformers[sentencepiece]>=4.26.0",
     "torch>=1.9",
+    "torchvision",
```

Comment:
why?

Reply:
Needed for preprocessing. 
I think this is not an issue as long as we have pytorch as a requirement. When we removed pytorch (which we should), we can handle this case as well.